tests/Digdir.Domain.Dialogporten.Application.Integration.Tests/Features/V1/ServiceOwner/Dialogs/Queries/GetDialogTests.cs

```diff
@@ -49,7 +49,13 @@ public async Task Get_ReturnsDialog_WhenDialogExists()
         result.Should().BeEquivalentTo(createCommand, options => options
```

Comment:
Denne bør skrives om til å oppgi ID-er for begge typer actions, attachments og transmission attachments, og ikke ekskludere dem fra sammenligningen

Reply:
Oppdaterte DialogGenerator til å Generere ID for Api/Gui actions og attachments

Reply:
>  .For(x => x.Attachments)
                .Exclude(x => x.Id)
            .For(x => x.ApiActions)
                .Exclude(x => x.Id)
            .For(x => x.GuiActions)
                .Exclude(x => x.Id));
                
Disse må fjernes.